server/src/main/java/org/elasticsearch/index/mapper/IdFieldMapper.java

```diff
@@ -48,6 +48,8 @@
 import org.elasticsearch.indices.breaker.CircuitBreakerService;
 import org.elasticsearch.search.DocValueFormat;
 import org.elasticsearch.search.MultiValueMode;
+import org.elasticsearch.search.aggregations.support.CoreValuesSourceType;
```

Comment:
I can't tell if github is being weird with the diff or if this is an actual change, but it seems these are listed below as well?  Surprised precommit doesn't fail this